src/app/controllers/ModifierController.ts

```diff
@@ -8,28 +8,11 @@ export interface ModifierChoiceInput {
   available: boolean;
   price: number;
 }
-export interface CreateModifierInput {
-  name: string;
-  identifier: string;
-  description: string;
-  required: boolean;
-  choices: Array<ModifierChoiceInput>;
-  default?: string;
-  default_choices: Array<string>;
+export interface CreateModifierInput extends Partial<Omit<Modifier, keyof DefaultControllerRequired >> {
```

Comment:
Partial is only on the deprecated field so you can just ignore the partial or only make that field partial by doing
Partial<Omit<Modifier, keyof defaultController> , 'default'>

Reply:
Partial only return <T>.

if i make Partial<Omit<Modifier, keyof defaultController> , 'default'>.. give a error.

Reply:
Just make default optional on Modifier, then it will propagate :)

Reply:
Partial<Omit<Modifier, keyof DefaultControllerRequired >, 'default'> 

making default?: string  on Modifier still showing a error.

Reply:
![image](https://user-images.githubusercontent.com/64808456/107292385-c2dde680-6a37-11eb-8c07-f38af773885d.png)

Reply:
@ralph-dev , take a look in the screeshoot. ?

Reply:
There is no error in the screenshot?